### PR TITLE
Add automation title to Analytics header [MAILPOET-5616]

### PR DESCRIPTION
**Pull request by websupporter**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets
[MAILPOET-5616]

## After-merge notes

_N/A_

## Tasks

- [x] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] I added sufficient test coverage
- [x] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5616]: https://mailpoet.atlassian.net/browse/MAILPOET-5616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx [hunk 1]
```diff
@@ -38,10 +43,38 @@ function Analytics(): JSX.Element {
...
+  if (automation.status === AutomationStatus.DEACTIVATING) {
+    status = __('Deactivating', 'mailpoet');
+    statusClass = 'mailpoet-analytics-badge-warning';
+  }
+
+  return (
+    <TopBarWithBeamer>
+      <p className="mailpoet-automation-analytics-title">
+        {__('Automations', 'mailpoet')} › <strong>{automation.name}</strong>
+        <div className={`mailpoet-analytics-badge ${statusClass}`}>
```
Comment (brezocordero) [inline mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx]:
```suggestion
        <div className={classNames('mailpoet-analytics-badge', statusClass)}>
```

File: mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx [hunk 2]
```diff
@@ -38,10 +43,38 @@ function Analytics(): JSX.Element {
...
+  }
+  if (automation.status === AutomationStatus.DEACTIVATING) {
+    status = __('Deactivating', 'mailpoet');
+    statusClass = 'mailpoet-analytics-badge-warning';
+  }
+
+  return (
+    <TopBarWithBeamer>
+      <p className="mailpoet-automation-analytics-title">
+        {__('Automations', 'mailpoet')} › <strong>{automation.name}</strong>
```
Comment (brezocordero) [inline mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx]: If it is a breadcrumb, I would expect the `{__('Automations', 'mailpoet')}` to be a link.
If it is a title, maybe use a heading tag for semantics?

**Review by brezocordero — COMMENTED** [review]
@websupporter 
works well, just a small suggestion on the classes and a question.

**Comment by websupporter** [comment]
Thank you @brezocordero, made the "Automations" a link which directs to the automations listing and used `classnames` now. I rebased with trunk and put it all in the same commit.

**Review by brezocordero — APPROVED** [review]
LGTM! Thanks for making the changes!

**Comment by veljkho** [comment]
Hi @websupporter 
I'm seeing blank page issue on Automations page when installed this build on Atomic site.
Please reach out to me on Slack for site credentials.

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**